dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -215,6 +215,16 @@ img.video_thumbnail {
   }
 }
 
+.sign_in.show_only_on_mobile {
+  float: right;
+  margin-right: 10px;
+  margin-bottom: 10px;
+
+  @media screen and (max-device-height: 900px) and (max-device-width: 500px) {
```

Comment:
Yeah. I decided to use the same conditions that we use for hiding the header, just so they would never appear both at the same time.